### PR TITLE
#164005365 SumUp array of numbers

### PR DESCRIPTION
**Pull request by VerbenaZ**
I have summed up array of numbers, such that array of numbers can be imputed in incremental or decreasing order and it returns the output of the summed arrays.

const SumUp = (okon, akpos) => {
const here = okon + akpos;
return here; 
}
console.log(SumUp (5, 15));

### CODE REVIEW

File: Level_1.js/SumUp.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const SumUp = [1, 2, 3, 4];
```
Comment (nwashangai) [inline Level_1.js/SumUp.js]: Please I don't want a constant as variable

File: Level_1.js/SumUp.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+const sum = SumUp. reduce ((total, value) => total + value, 0);
```
Comment (nwashangai) [inline Level_1.js/SumUp.js]: put this in a function and call the function sumUp, something like this 
```
const sumUp = (arr) => {
return arr.reduce ((total, value) => total + value, 0);
}
```

**Review by nwashangai — CHANGES_REQUESTED** [review]
Good attempt.

**Comment by VerbenaZ** [comment]
i have done the corrections, it no longer sum up to 10, but rather sums varieties of arrays, no matter the numbers imputed and returns the total of the summed arrays.

const SumUp = (okon, akpos) => {
const here = okon + akpos;
return here; 
}
console.log(SumUp (5, 15));

**Pull request closed without merging**